### PR TITLE
stolostron/backlog#19300 upd network

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by thuyn-581** [comment]
lgtm

**Review by thuyn-581 — COMMENTED** [review]
lgtm

**Review by thuyn-581 — APPROVED** [review]
lgtm

**Review by swopebe — CHANGES_REQUESTED** [review]
peer review

File: credentials/credential_bare.adoc [hunk 1]
```diff
@@ -14,8 +14,7 @@ You need a credential to use the {product-title} console to deploy and manage a
 
 You need the following prerequisites before creating a credential:
 
-* A {product-title} hub cluster that is deployed.
-When managing bare metal clusters, you must have the hub cluster installed on {ocp} version 4.6 or later.
+* A {product-title} hub cluster that is deployed. When managing bare metal clusters, you must have the hub cluster installed on {ocp} version 4.6 or later.
```
Comment (dockerymick) [inline credentials/credential_bare.adoc:17]: It seems like the statements are two separate prereqs. 

* A {product-title} hub cluster that is deployed.
* You must have the hub cluster installed on {ocp} version 4.6 or later when you manage bare metal clusters. 

Let me know your thoughts or if i am misunderstanding

**Review by dockerymick — APPROVED** [review]
left a comment, otherwise this looks good to me! Format is aligned well and links work well! 
Screen Shot 2022-07-15 at 10 50 13 AM

File: credentials/credential_bare.adoc (hunk 1 above)
Comment (cadawson) [reply credentials/credential_bare.adoc:17]: I like your change. Thanks!

**Pull request merged**